components/forecast/Daily.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import moment from 'moment';
+
+import weatherIcons from '../../constants/weathericons';
+import {ForecastTimestep} from '../../utils/locationforecast';
+
+function maxTemp(forecast: ForecastTimestep | undefined): number | undefined {
+  if (!forecast) {
+    return;
+  }
+
+  const first = forecast.data.next_1_hours?.details?.air_temperature_max;
+  const second = forecast.data.next_6_hours?.details?.air_temperature_max;
+  const third = forecast.data.next_12_hours?.details?.air_temperature_max;
+  if (!first || !second || !third) {
+    return undefined;
+  }
+
+  return Math.max(first, second, third);
+}
+
+function minTemp(forecast: ForecastTimestep | undefined): number | undefined {
+  if (!forecast) {
+    return;
+  }
+
+  const first = forecast.data.next_1_hours?.details?.air_temperature_min;
+  const second = forecast.data.next_6_hours?.details?.air_temperature_min;
+  const third = forecast.data.next_12_hours?.details?.air_temperature_min;
+  if (!first || !second || !third) {
+    return undefined;
+  }
+
+  return Math.min(first, second, third);
+}
+
+function maxPrecipitation(
```

Comment:
Regarding the comment on temperature, in this case it is slightly different if you apply the change that I suggested: It is not possible to sum n-hour precipitation max, and expect to get a correct result - that would cause over-forecasting of rainfall.

The best that can be done in this case is to simply use the values as they are.

Reply:
@vegardb, I do not understand your recommendation. Mind dumbing it down?

Reply:
Essentially this is the case:

```
24hourMaxPrecipitation < sum(6hourMaxPrecipitation values)
```

However, I see that `precipitation_amount_max` is not available for areas outside of the nordic regions, so you will never get any real data for this anyway.

You may sum `precipitation_amount`, though. That should give a correct result for expected (not max expected) 
precipitation.